_ignite-app-registry/explorer/app.json

```diff
@@ -0,0 +1,49 @@
+{
+    "appName": "explorer",
+    "appDescription": "Shows real time feed of your blockchain.",
+    "latestRelease": {
+      "version": "0.2",
+      "date": "2024-02-27",
+      "releaseNotesUrl": "https://github.com/ignite/apps/releases/tag/explorer%2Fv0.2.0"
+    },
+    "igniteCompatibility": {
+      "lastTestedVersion": "ignite v28.2.0"
```

Comment:
I think it should just be the version

Reply:
We'll use version like defined in: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#dependencies